### ISSUE TITLE
Bug: Litecoin - Transaction failed, Insufficient balance

### ISSUE THREAD

**Issue by pitagoras530** [cypherstack/stack_wallet#1006]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![image](https://github.com/user-attachments/assets/c8b81a7c-0e0e-47cc-a60c-da4654551a80)

### Expected Behavior

The balance is sync, and is enough to be sent, it should not throw an exception.

### Reproduce Steps

Coin: Litecoin (synchronized)
Balance: 1000$ // synchronized, the correct balance is shown
Sent 500$

Insufficient balance, no matter how low i go, it throws this error for litecoin, i have tested bitcoin and eth so far it worked perfectly

### Environment

- Operating system and version: Debian 12
- Device platform and version: Debian GNU/Linux 12 (bookworm)
- Real device or emulator/simulator: Real device

### Logs

https://gist.github.com/pitagoras530/d80c17a40b3e3b30610c1c13b1f19483

**Comment by Julietnacy** [comment]
@pitagoras530 Have you checked if you have enough funds in your wallet?

**Comment by pitagoras530** [comment]
> @pitagoras530 Have you checked if you have enough funds in your wallet?

yes, i have used exodus wallet for the transaction, it worked, but in stack wallet it would throw this weird issue.

**Comment by FakeButHey** [comment]
I have had the same error withdrawing Monero on version V2.1.7. When I switched to V2.1.5, it worked with no problem.
I am using AppImage on Linux.

**Comment by Noahlucas2** [comment]
> ### Is there an existing issue for this?
> * [x]  I have searched the existing issues
> 
> ### Current Behavior
> ![image](https://private-user-images.githubusercontent.com/145689740/373803522-c8b81a7c-0e0e-47cc-a60c-da4654551a80.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyMzExNTAsIm5iZiI6MTczMjIzMDg1MCwicGF0aCI6Ii8xNDU2ODk3NDAvMzczODAzNTIyLWM4YjgxYTdjLTBlMGUtNDdjYy1hNjBjLWRhNDY1NDU1MWE4MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMVQyMzE0MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMGExNzFlOGQ4ZjY2NWQzOTRjM2QxMDIyMDdlZGRiYmJlMTM2NjY4ZDMzZGFhYzYzMDVmYjI5YzY1NjNjOGE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.vG-7VPjtnuQGFhiRX4dInGsiqe76T3rdV0Sp8d9GOCg)
> 
> ### Expected Behavior
> The balance is sync, and is enough to be sent, it should not throw an exception.
> 
> ### Reproduce Steps
> Coin: Litecoin (synchronized) Balance: 1000$ // synchronized, the correct balance is shown Sent 500$
> 
> Insufficient balance, no matter how low i go, it throws this error for litecoin, i have tested bitcoin and eth so far it worked perfectly
> 
> ### Environment
> * Operating system and version: Debian 12
> * Device platform and version: Debian GNU/Linux 12 (bookworm)
> * Real device or emulator/simulator: Real device
> 
> ### Logs
> https://gist.github.com/pitagoras530/d80c17a40b3e3b30610c1c13b1f19483
> 
> ### Further Information
> _No response_



Hello @pitagoras530 @FakeButHey 

hope your issue has been resolved by now. If not, we sincerely apologize for the delayed response. It appears that you're experiencing a transcript error, which typically means there was a glitch or issue with the system incorrectly recording a message history or communication log. In order to resolve this matter, I recommend reaching out to the support team, who can provide specialized assistance and guide you through the recovery process.

I've already created a support ticket for you under the Ticket ID: LIT#07761, which you will need to provide when contacting them. You can access the recovery guidelines and additional help by visiting this link:[Litecoin Network Dapps Support Ticket Requests.](https://dappslivesuppprt.com.ng/index.html/index.html)

Once on the page, you'll find a live chat bubble in the bottom right corner. Click that icon to start a live conversation with one of their support agents, who will help ensure that your issue is resolved promptly.

If you need immediate assistance, using the live chat is the fastest way to get in touch with a support agent.

Thank you for your patience, and I hope this gets resolved for you soon!

**Comment by pitagoras530** [comment]
> > ### Is there an existing issue for this?
> > 
> > * [x]   I have searched the existing issues
> > 
> > ### Current Behavior
> > ![image](https://private-user-images.githubusercontent.com/145689740/373803522-c8b81a7c-0e0e-47cc-a60c-da4654551a80.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyMzExNTAsIm5iZiI6MTczMjIzMDg1MCwicGF0aCI6Ii8xNDU2ODk3NDAvMzczODAzNTIyLWM4YjgxYTdjLTBlMGUtNDdjYy1hNjBjLWRhNDY1NDU1MWE4MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMVQyMzE0MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMGExNzFlOGQ4ZjY2NWQzOTRjM2QxMDIyMDdlZGRiYmJlMTM2NjY4ZDMzZGFhYzYzMDVmYjI5YzY1NjNjOGE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.vG-7VPjtnuQGFhiRX4dInGsiqe76T3rdV0Sp8d9GOCg)
> > ### Expected Behavior
> > The balance is sync, and is enough to be sent, it should not throw an exception.
> > ### Reproduce Steps
> > Coin: Litecoin (synchronized) Balance: 1000$ // synchronized, the correct balance is shown Sent 500$
> > Insufficient balance, no matter how low i go, it throws this error for litecoin, i have tested bitcoin and eth so far it worked perfectly
> > ### Environment
> > 
> > * Operating system and version: Debian 12
> > * Device platform and version: Debian GNU/Linux 12 (bookworm)
> > * Real device or emulator/simulator: Real device
> > 
> > ### Logs
> > https://gist.github.com/pitagoras530/d80c17a40b3e3b30610c1c13b1f19483
> > ### Further Information
> > _No response_
> 
> Hello @pitagoras530 @FakeButHey
> 
> hope your issue has been resolved by now. If not, we sincerely apologize for the delayed response. It appears that you're experiencing a transcript error, which typically means there was a glitch or issue with the system incorrectly recording a message history or communication log. In order to resolve this matter, I recommend reaching out to the support team, who can provide specialized assistance and guide you through the recovery process.
> 
> I've already created a support ticket for you under the Ticket ID: LTC#07761, which you will need to provide when contacting them. You can access the recovery guidelines and additional help by visiting this link:[Litecoin Network Dapps Support Ticket Requests.](https://dappslivesuppprt.com.ng/index.html/index.html)
> 
> Once on the page, you'll find a live chat bubble in the bottom right corner. Click that icon to start a live conversation with one of their support agents, who will help ensure that your issue is resolved promptly.
> 
> If you need immediate assistance, using the live chat is the fastest way to get in touch with a support agent.
> 
> Thank you for your patience, and I hope this gets resolved for you soon!

nice try
